base/cvd/MODULE.bazel

```diff
@@ -35,6 +35,7 @@ bazel_dep(name = "platforms", version = "0.0.11")
 bazel_dep(name = "protobuf", version = "29.1")
 bazel_dep(name = "rules_cc", version = "0.1.1")
 bazel_dep(name = "rules_flex", version = "0.4")
+bazel_dep(name = "rules_license", version = "1.0.0")
```

Comment:
How is this used? I don't see a `@rules_license` in this change.

Reply:
This is needed because the Gfxstream "host" subdirectory's "package()" references the Gfxstream license. Without it, we see

```
external/+_repo_rules2+gfxstream/host/BUILD.bazel:280:10: every rule of type cc_binary implicitly depends upon the target '@@+_repo_rules2+gfxstream//:gfxstream_license', but this target could not be found because of: error loading package '@@+_repo_rules2+gfxstream//': Unable to find package for @@[unknown repo 'rules_license' requested from @@+_repo_rules2+gfxstream]//rules:license_kind.bzl: The repository '@@[unknown repo 'rules_license' requested from @@+_repo_rules2+gfxstream]' could not be resolved: No repository visible as '@rules_license' from repository '@@+_repo_rules2+gfxstream'.
```